cds/modules/theme/assets/bootstrap3/js/cds_records/cdsRecord.js

```diff
@@ -188,9 +190,24 @@ function cdsRecordController($scope, $sce, $http, $timeout, $filter) {
   $scope.$watch("record", function (newVal) {
     if (newVal) {
       $scope.initVttLoad(newVal);
+      $scope.prepareDescriptions(newVal.metadata.description);
     }
   });
 
+  $scope.prepareDescriptions = function (description) {
+    if (!description) {
+      $scope.shortDescription = "No description";
+      $scope.fullDescription = "No description";
+      return;
+    }
+
+    const lines = description.split(/\r?\n/);
+    const firstTen = lines.slice(0, 10).join("\n");
```

Comment:
minor: Maybe extract the number in a constant so we can easily change if needed?